src/gobprepare/importers/dump_importer.py

```diff
@@ -0,0 +1,135 @@
+import contextlib
+from io import StringIO
+import tempfile
+import gzip
+import re
+from typing import Iterator, Optional
+
+from gobprepare.importers.typing import ReadConfig, SqlDumpImporterConfig
+from gobconfig.datastore.config import get_datastore_config
+from gobcore.datastore.factory import DatastoreFactory
+from gobcore.datastore.objectstore import ObjectDatastore
+from gobcore.datastore.sql import SqlDatastore
+from gobcore.logging.logger import logger
+
+from gobcore.exceptions import GOBException
+
+@contextlib.contextmanager
+def _load_from_objectstore(datastore: ObjectDatastore) -> Iterator[str]:
+    datastore.connect()
+    try:
+        obj_info = next(datastore.query(None), None)
```

Comment:
The extra argument in next() is the default value when it reaches the end of the iterator, this will not raise a StopIteration